returnn/datasets/postprocessing.py

```diff
@@ -31,8 +45,14 @@ class PostprocessingDataset(CachedDataset2):
     SpecAugment or speed perturbation into the data loading pipeline.
 
     The integration into the data loading pipeline makes it easy to distribute the
-    data processing work across multiple CPU cores using `MultiProcDataset` and in
-    turn frees the GPU from data preprocessing tasks.
+    data processing work across multiple CPU cores and in turn frees the GPU from
+    data preprocessing tasks.
+
```

Comment:
If I understand correctly, the former (`MultiProcDatase`) uses multi-processing and the latter (with `num_workers`) uses multi-_threading_, so "Multiprocessing can either be done using ..." is misleading?

Reply:
Correct!